assets/src/models/finishedDetour.ts

```diff
@@ -16,6 +17,7 @@ export const FinishedDetourData = type({
     detour: array(Coordinate),
     after_detour: array(Coordinate),
   }),
+  detour_shape: DetourShapeData,
```

Comment:
I feel like this interface and this type have gotten kind of _messy_, but I almost would rather clean that up when we make a real `/detours` API.